torch/csrc/jit/codegen/cuda/test/test_gpu_validator.h

```diff
@@ -398,7 +398,8 @@ void testValidate(
           aten_output_tensor.allclose(
               fusion_output_tensor.to(aten_output_tensor.dtype()),
               tolerance_values.second,
-              tolerance_values.first),
+              tolerance_values.first,
+              /*equal_nan=*/true),
```

Comment:
It's expected that `acosh`, `erfinv`, `erfcinv` return `nan` for certain inputs.
Other tests might rely on the fact that by default nans are not equal. I'll update here if I see a failing test.

Reply:
Currently seeing
```
[  FAILED  ] NVFuserTest.FusionBiasGeluBwd_CUDA // fails with or without equal_nan=true
[  FAILED  ] NVFuserTest.FusionWelfordPersistence_CUDA  // passes when run individually independent of equal_nan value
[  FAILED  ] NVFuserTest.FusionScheduleTransposeMultipleInputOutput_CUDA // passes when run individually independent of equal_nan value
```
So it seems `uqual_nan=true` is harmless.